src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -394,4 +414,8 @@ public DateTime getReturnDate() {
   public void changeItemStatus(String itemStatus) {
     representation.put(LoanProperties.ITEM_STATUS, itemStatus);
   }
+
+  public void changeDeclaredLostDateTime(DateTime dateTime) {
+    write(representation, LoanProperties.DECLARED_LOST_DATE, dateTime);
+  }
```

Comment:
mod-circulation is trying to get away from directly mutating the underlying JSON within the domain entities, preferring explicit fields that are mapped to storage or external representations further out to the edges.

If possible, I'd prefer this approach was taken here, so as to not increase the dependence upon the internal JSON, which I would like to remove at some point (however I do not have the bandwidth to do so at the moment).